inngest/_internal/function.py

```diff
@@ -124,7 +124,9 @@ def on_failure_fn_id(self) -> str | None:
     def __init__(
         self,
         opts: FunctionOpts,
-        trigger: function_config.TriggerCron | function_config.TriggerEvent,
+        triggers: list[
```

Comment:
This is a breaking change since `Function` is public:
https://github.com/inngest/inngest-py/blob/main/inngest/__init__.py#L30

Should be OK to keep it as `trigger`, even though it's a misnomer when multiple triggers are used

Reply:
@goodoldneon Ah, gotcha. Will change this back - figured they were internal. 🙂